mlos_core/mlos_core/optimizers/flaml_optimizer.py

```diff
@@ -137,7 +150,7 @@ def _target_function(self, config: dict) -> Union[dict, None]:
         """
         cs_config = normalize_config(self.optimizer_parameter_space, config)
         if cs_config in self.evaluated_samples:
-            return {'score': self.evaluated_samples[cs_config].score}
+            return {self._optimization_targets[0]: self.evaluated_samples[cs_config].score}
```

Comment:
Nit: as a maintenance thing, for FLAML, I'd suggest we introduce a `self._optimization_target` property that returns `self._optimization_targets[0]` so that if they ever change it in the future it'll be a little easier to find the places to adjust.

Reply:
agree